<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugDialog</class>
 <widget class="QWidget" name="debugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Monitor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string notr="true">Write thread</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListWidget" name="writeThreadList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string notr="true">Transfer list</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="tranferList"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="readStatus">
       <property name="text">
        <string notr="true">Read thread running</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="readBlocking">
       <property name="text">
        <string notr="true">Block to read acces</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
